### PR TITLE
[WIP] Extracting MFTF tests in separate test files

### PR DESCRIPTION
**Pull request by eduard13**
### Description (*)

This PR extracts the MFTF tests into separate test files based on the [MFTF Best Practices](https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices) for the following modules:
- Bundle
- Captcha
- ConfigurableProduct
- Customer
- Tax

Additionally, it also replaces the admin logging out `<amOnPage url="{{AdminLogoutPage.url}}" stepKey="logout"/>` action by `AdminLogoutActionGroup` action group.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by lbajsarowicz** [comment]
@eduard13 awesome! 
Keep going with other modules!

File: app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductAddConfigurationTest.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <title value="Admin should be able to edit configuration to add a value to an existing attribute"/>
+            <description value="Admin should be able to edit configuration to add a value to an existing attribute"/>
+            <severity value="AVERAGE"/>
+            <testCaseId value="MC-95"/>
+            <group value="ConfigurableProduct"/>
+        </annotations>
+
+        <before>
+            <createData entity="ApiCategory" stepKey="createCategory"/>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductAddConfigurationTest.xml]: Use `AdminLoginActionGroup` instead.

File: app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductBulkDeleteTest.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            <createData entity="ConfigurableProductAddChild" stepKey="createConfigProductAddChild5">
+                <requiredEntity createDataKey="createProduct3"/>
+                <requiredEntity createDataKey="createConfigChildProduct5"/>
+            </createData>
+            <createData entity="ConfigurableProductAddChild" stepKey="createConfigProductAddChild6">
+                <requiredEntity createDataKey="createProduct3"/>
+                <requiredEntity createDataKey="createConfigChildProduct6"/>
+            </createData>
+
+            <actionGroup ref="LoginAsAdmin" stepKey="login"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductBulkDeleteTest.xml]: Use `AdminLoginActionGroup` instead

File: app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductDisableAnOptionTest.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        </after>
+
+        <!--check storefront for both options-->
+        <amOnPage url="$$createConfigProduct.sku$$.html" stepKey="amOnStorefront1"/>
+        <waitForPageLoad stepKey="wait1"/>
+        <see selector="{{StorefrontProductInfoMainSection.productAttributeOptions1}}" userInput="option1" stepKey="seeOption1Storefront"/>
+        <see selector="{{StorefrontProductInfoMainSection.productAttributeOptions1}}" userInput="option2" stepKey="seeOption2Storefront"/>
+
+        <!--go to admin and disable an option-->
+        <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/ConfigurableProduct/Test/Mftf/Test/AdminConfigurableProductDisableAnOptionTest.xml]: Use `AdminLoginActionGroup` instead

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Could you also replace all the occurences of `LoginAsAdmin` in changed files to `AdminLoginActionGroup`?

Thanks!

**Comment by lbajsarowicz** [comment]
Don't change the names unless my PRs related to change Test names is merged, please.

**Comment by eduard13** [comment]
Agreed, thank you for letting me know.

**Comment by lbajsarowicz** [comment]
@eduard13 The PRs I've mentioned:
* https://github.com/magento/magento2/pull/27118
* https://github.com/magento/magento2-page-builder/pull/399
* https://github.com/magento/partners-magento2ee/pull/151
* https://github.com/magento/partners-magento2b2b/pull/54
* https://github.com/magento/inventory/pull/2857

**Comment by eduard13** [comment]
Hi @lbajsarowicz, I'd like to proceed with the other modules in a separate PR, in order to provide these changes is a smaller bunches (also if there will be some conflicts, will be easier to manage them). What do you think?
Also, the failing tests don't look to be somehow related with the changes of this PR.

**Comment by lbajsarowicz** [comment]
@eduard13 doing changes per module is the easiest way of doing major changes. 

My changes to action groups were waiting 7 months for review (single pr) so I split them per module and in 3 months - were merged 😀

The failed tests are not related to your changes

**Pull request closed without merging**